python/semantic_kernel/connectors/memory/mongodb_atlas/mongodb_atlas_memory_store.py

```diff
@@ -268,7 +268,7 @@ async def get_nearest_matches(
         if min_relevance_score:
             pipeline.append({"$match": {"$gte": ["$score", min_relevance_score]}})
 
-        cursor: MotorCommandCursor = self.database[collection_name].aggregate(pipeline)
+        cursor: core.AgnosticCommandCursor = self.database[collection_name].aggregate(pipeline)
```

Comment:
Is this type annotation even needed? What happens if we remove it?

Reply:
Honestly, it should be perfectly fine if we just remove it outright.